pfio/v2/fs.py

```diff
@@ -67,28 +67,15 @@ def __repr__(self):
         return str(self.__str__())
 
 
-class ForkedError(RuntimeError):
```

Comment:
As stated in the HDFS section, I think this class should be preserved for the real use. Also, it should be preserved for a while in case it's imported in applications' code.